I see some errors in the console, but ultimately things are passing:

```
3: 2022-05-23 18:35:51+0000 [-] [rosbridge_websocket-1] Failed to publish log message to rosout: publisher's context is invalid, at ./src/rcl/publisher.c:389
[4992](https://github.com/RobotWebTools/rosbridge_suite/runs/6560941502?check_suite_focus=true#step:4:5003)
  3: 2022-05-23 18:35:51+0000 [-] [rosbridge_websocket-1] Traceback (most recent call last):
[4993](https://github.com/RobotWebTools/rosbridge_suite/runs/6560941502?check_suite_focus=true#step:4:5004)
  3: 2022-05-23 18:35:51+0000 [-] [rosbridge_websocket-1]   File "/home/runner/work/rosbridge_suite/rosbridge_suite/ros_ws/install/rosbridge_server/lib/rosbridge_server/rosbridge_websocket", line 332, in <module>
[4994](https://github.com/RobotWebTools/rosbridge_suite/runs/6560941502?check_suite_focus=true#step:4:5005)
  3: 2022-05-23 18:35:51+0000 [-] [rosbridge_websocket-1]     main()
[4995](https://github.com/RobotWebTools/rosbridge_suite/runs/6560941502?check_suite_focus=true#step:4:5006)
  3: 2022-05-23 18:35:51+0000 [-] [rosbridge_websocket-1]   File "/home/runner/work/rosbridge_suite/rosbridge_suite/ros_ws/install/rosbridge_server/lib/rosbridge_server/rosbridge_websocket", line 327, in main
[4996](https://github.com/RobotWebTools/rosbridge_suite/runs/6560941502?check_suite_focus=true#step:4:5007)
  3: 2022-05-23 18:35:51+0000 [-] [rosbridge_websocket-1]     rclpy.shutdown()
[4997](https://github.com/RobotWebTools/rosbridge_suite/runs/6560941502?check_suite_focus=true#step:4:5008)
  3: 2022-05-23 18:35:51+0000 [-] [rosbridge_websocket-1]   File "/opt/ros/rolling/local/lib/python3.10/dist-packages/rclpy/__init__.py", line 126, in shutdown
[4998](https://github.com/RobotWebTools/rosbridge_suite/runs/6560941502?check_suite_focus=true#step:4:5009)
  3: 2022-05-23 18:35:51+0000 [-] [rosbridge_websocket-1]     _shutdown(context=context)
[4999](https://github.com/RobotWebTools/rosbridge_suite/runs/6560941502?check_suite_focus=true#step:4:5010)
  3: 2022-05-23 18:35:51+0000 [-] [rosbridge_websocket-1]   File "/opt/ros/rolling/local/lib/python3.10/dist-packages/rclpy/utilities.py", line 58, in shutdown
[5000](https://github.com/RobotWebTools/rosbridge_suite/runs/6560941502?check_suite_focus=true#step:4:5011)
  3: 2022-05-23 18:35:51+0000 [-] [rosbridge_websocket-1]     return context.shutdown()
[5001](https://github.com/RobotWebTools/rosbridge_suite/runs/6560941502?check_suite_focus=true#step:4:5012)
  3: 2022-05-23 18:35:51+0000 [-] [rosbridge_websocket-1]   File "/opt/ros/rolling/local/lib/python3.10/dist-packages/rclpy/context.py", line 102, in shutdown
[5002](https://github.com/RobotWebTools/rosbridge_suite/runs/6560941502?check_suite_focus=true#step:4:5013)
  3: 2022-05-23 18:35:51+0000 [-] [rosbridge_websocket-1]     self.__context.shutdown()
[5003](https://github.com/RobotWebTools/rosbridge_suite/runs/6560941502?check_suite_focus=true#step:4:5014)
  3: 2022-05-23 18:35:51+0000 [-] [rosbridge_websocket-1] rclpy._rclpy_pybind11.RCLError: failed to shutdown: rcl_shutdown already called on the given context, at ./src/rcl/init.c:241
[5004](https://github.com/RobotWebTools/rosbridge_suite/runs/6560941502?check_suite_focus=true#step:4:5015)
  Error: -05-23 18:35:52+0000 [-] [ERROR] [rosbridge_websocket-1]: process has died [pid 24266, exit code 1, cmd '/home/runner/work/rosbridge_suite/rosbridge_suite/ros_ws/install/rosbridge_server/lib/rosbridge_server/rosbridge_websocket --ros-args --params-file /tmp/launch_params_ddocqej6'].
[5005](https://github.com/RobotWebTools/rosbridge_suite/runs/6560941502?check_suite_focus=true#step:4:5016)
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -25,6 +25,8 @@ jobs:
             os: ubuntu-20.04
           - ros: galactic
             os: ubuntu-20.04
+          - ros: humble
+            os: ubuntu-22.04
           - ros: rolling
             os: ubuntu-22.04
 

